src/components/sections/ProductGallery/ProductGallery.tsx

```diff
@@ -23,42 +22,38 @@ interface Props {
 }
 
 function ProductGallery({ title, slug }: Props) {
+  const [isFilterOpen, setIsFilterOpen] = useState<boolean>(false)
   const { pages, state: searchState, addNextPage, addPrevPage } = useSearch()
   const { data } = useGalleryQuery()
-
   const totalCount = useTotalCount(data)
-
-  const { next, prev } = usePagination(totalCount)
-  const { isMobile } = useWindowDimensions()
-
-  const [isFilterOpen, setIsFilterOpen] = useState<boolean>(false)
-
   const orderedFacets = useOrderedFacets(data)
+  const { next, prev } = usePagination(totalCount)
 
   return (
-    <>
-      <div className="product-listing__filters">
-        <Filter
-          slug={slug}
-          isOpen={isFilterOpen}
-          facets={orderedFacets}
-          onDismiss={() => setIsFilterOpen(false)}
-        />
-      </div>
+    <div className="product-listing / grid-content-full">
+      <div className="product-listing__content-grid / grid-content">
+        <div className="product-listing__filters">
+          <Filter
+            slug={slug}
+            isOpen={isFilterOpen}
+            facets={orderedFacets}
+            onDismiss={() => setIsFilterOpen(false)}
+          />
+        </div>
 
-      <div
-        className="product-listing__results-count"
-        data-testid="total-product-count"
-        data-count={totalCount}
-      >
-        <h2>{totalCount} Results</h2>
-      </div>
+        <div
+          className="product-listing__results-count"
+          data-testid="total-product-count"
+          data-count={totalCount}
+        >
+          <h2>{totalCount} Results</h2>
+        </div>
 
-      <div className="product-listing__sort">
-        <Sort />
+        <div className="product-listing__sort">
+          <Sort />
 
-        {isMobile && (
           <Button
+            className="button display-mobile"
```

Comment:
This is not working as we expected 😢
 
![image](https://user-images.githubusercontent.com/3356699/154180421-173c7699-c48a-4418-8264-a711f894542c.png)

Reply:
Lemme try to find a solution